From 5c8745f71be16f0c394fb237e276032ab14de0b7 Mon Sep 17 00:00:00 2001
From: Luwei Zhou <b45643@freescale.com>
Date: Wed, 3 Dec 2014 16:21:40 +0800
Subject: [PATCH 0989/1543] MLK-9946 ARM: dts: Add new dts for MLB support for
 i.MX6SX-17x17-arm2 platform.

commit 49e3472de4237378ad0d47ae443981e478972d51 from
git://git.freescale.com/imx/linux-2.6-imx.git

Add new device tree  for MLB support for i.MX6SX-17x17-arm2 platform.

Signed-off-by: Luwei Zhou <b45643@freescale.com>
---
 arch/arm/boot/dts/imx6sx-17x17-arm2-mlb.dts |   18 ++++++++++++++++++
 1 files changed, 18 insertions(+), 0 deletions(-)
 create mode 100644 arch/arm/boot/dts/imx6sx-17x17-arm2-mlb.dts

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2-mlb.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2-mlb.dts
new file mode 100644
index 0000000..1a64bec
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2-mlb.dts
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2014 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sx-17x17-arm2.dts"
+
+&mlb {
+	status = "okay";
+};
+
+&usdhc2 {
+	/* pin conflict with mlb */
+	status = "disabled";
+};
-- 
1.7.5.4

